Week1/homework/js-exercises/typeChecker.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+var a = "fish in the see";
+var b = "lion in the den";
+
+var c = {
+    alpha :"purple",
+    beta:"yellow",
+    theta:"cyan"
+};
+
+var d = {car:"jaguar", color:"latte"};
+
```

Comment:
👍 for logic but again using `const` here instead of `var` would be nicer. Also, indentation for objects `c` and `d` are different even though both are objects. I would come back to my point of code readability and how properly writing code is also important. Could you please correct this?